curator-recipes/src/main/java/org/apache/curator/framework/recipes/locks/LockSchema.java

```diff
@@ -0,0 +1,22 @@
+package org.apache.curator.framework.recipes.locks;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import com.google.common.collect.Sets;
+
+public class LockSchema {
+    private final Set<String> paths;
+
+    public LockSchema() {
+        paths = new HashSet<String>();
+    }
+
+    public LockSchema(Set<String> paths) {
+        this.paths = Sets.newHashSet(paths);
+    }
+
+    public Set<String> getPaths() {
+        return Sets.newHashSet(paths);
+    }
+}
```

Comment:
This doesn't look like it adds a lot of value as a concrete class.  I would either just pass the set directly to the reaper, or else turn this into an interface.  You could also use Iterable instead of Set since you don't really depend on Set semantics.